.github/workflows/php.yml

```diff
@@ -57,6 +59,12 @@ jobs:
       env:
         PACKAGES: ${{ matrix.packages}}
 
+    - name: Check codestyle
+      run: |
+        mkdir -p tools/php-cs-fixer
+        composer require --working-dir=tools/php-cs-fixer friendsofphp/php-cs-fixer
```

Comment:
better keep this as `require-dev` dependency. this will allow every developer use this fixer later

Reply:
It was my first implementation of adding php-cs-fixer. But in case of adding fixer directly to project's depencencies there is the problem with test case "symfony 4.0.*". Suddenly php-cs-fixer conflicts with hhvm version that is used in github actions. Here it is: https://github.com/lamoda/php-metrics/runs/2314067832?check_suite_focus=true

Reply:
Hm, there is a problem with latest composers

https://github.com/composer/composer/issues/9301

Can we drop this section for now (but keep the `.php_cs`) ? I will fix it later

Reply:
I removed it